src/sass/base/_utilities.scss

```diff
@@ -0,0 +1,15 @@
+.mb-sm {
+  margin-bottom: 2rem;
+}
+
+.mb-md {
+  margin-bottom: 3rem;
+}
+
+.mb-lg {
+  margin-bottom: 4rem;
+}
+
+.mb-hg {
+  margin-bottom: 8rem;
+}
```

Comment:
should checkpout spacing sytem of exisiting libbraries/frameworks 

Eg: https://tailwindcss.com/docs/customizing-spacing

Reply:
Changed the spacing system to be in multiples of 2 and added media queries.

![image](https://github.com/user-attachments/assets/1f6fcbf0-0872-457d-b58d-0a6b617c5eea)

Reply:
As you are using rem this will be 20, 40, 60 px. Is this what you want?

Did you check how tailwind or other libraries are implemented? You should add spacing and names in a way that they are scalable (meaning if needed you can add more easily later). Right now b/w 20 and 40 there is a huge gap. If you want to add 2-3 spaing b/w them then how will you name them? The current names does not look to be scalable

Reply:
Also do you really need hudge spacing likes 20, 40, 60. No small spacing requirements like 4, 8 12 px?

Reply:
Read more:
https://atlassian.design/foundations/spacing
https://www.designsystems.com/space-grids-and-layouts/
https://carbondesignsystem.com/elements/spacing/overview/

Reply:
updated the space variables [6e92b61](https://github.com/Pranav-Patani/BookWise/pull/6/commits/6e92b6129886c9802c7f8aeaf4b75a655c16d4ea)